README.md

```diff
@@ -31,8 +31,7 @@ Write containerd defaults to the containerd config.toml file.
 Set systemd as cgroup driver in config.toml. Only valid with `containerd_config_default_write: true`
 
     docker_apt_release_channel: stable
-    docker_apt_arch: '{{ (ansible_architecture == "aarch64") | ternary("arm64", "amd64") }}'
```

Comment:
Is the arch not required anymore for differentiating between AMD64 and ARM64?

Reply:
I just tested the role within an debian-arm64 VM and installed containerd successfully. According to https://www.reddit.com/r/debian/comments/1jqdgrz/apt_modernizesources_removes_archamd64_part/ the default architecture is automatically determined.